<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>881</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>160</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_any">
       <property name="text">
        <string>pushButton_any</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_all">
       <property name="text">
        <string>pushButton_all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_none">
       <property name="text">
        <string>pushButton_none</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_count">
       <property name="text">
        <string>pushButton_count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_getsize">
       <property name="text">
        <string>pushButton_getsize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_test">
       <property name="text">
        <string>pushButton_test</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_set">
       <property name="text">
        <string>pushButton_set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="text">
        <string>pushButton_reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ul">
       <property name="text">
        <string>pushButton_ul</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_flip">
       <property name="text">
        <string>pushButton_flip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ull">
       <property name="text">
        <string>pushButton_ull</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <zorder>spinBox</zorder>
   <zorder>lineEdit</zorder>
   <zorder>textBrowser</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>checkBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
